backend/tests/integration/test_events_api.py

```
@@ -1986,3 +1986,529 @@ def test_get_standalone_deadline_entry_shows_no_series(
+                ticket_required=True,
+                ticket_status="purchased",
+            ),
+        ]
+
+        test_db_session.add_all(events)
+        test_db_session.commit()
+        for e in events:
+            test_db_session.refresh(e)
+        return events
+
+    def test_preset_upcoming_30d(self, test_client, preset_events):
```

Ruff (0.15.0)

[warning] Unused method argument: `preset_events`

(ARG002)